### PR TITLE
fix(cache-missed): remove extended_bounds so that the query always hits the cache

### PR DESCRIPTION
**Pull request by orphaner**
Hello,

I propose you this PR I've tested with our OPS.
The details are in the issue gravitee-io/issues#1545

Nicolas

### CODE REVIEW

**Comment by brasseld** [comment]
Hi @orphaner,

Please reword your commit with the `fix(analytics)` prefix.
Also please include a reference to the github issue in your commit.

**Comment by orphaner** [comment]
Hy @brasseld 
Done !

**Comment by brasseld** [comment]
Ok, please reword it to:

```
fix(analytics): remove extended_bounds so that the query always hits the cache

Closes gravitee-io/issues#1545
```

Sorry, it just to preserve commit coherency.

Thanks for your understanding !

**Comment by orphaner** [comment]
> Ok, please reword it to:
> 
> ```
> fix(analytics): remove extended_bounds so that the query always hits the cache
> 
> Closes gravitee-io/issues#1545
> ```
> 
> Sorry, it just to preserve commit coherency.
> 
> Thanks for your understanding !

No problem, I understand very well.
Nicolas

**Comment by brasseld** [comment]
A really simple fix but really efficient ;)

Thanks a lot!
